hypothesis/client
# Make sure export suggested filename is updated when group changes

This PR ensures export annotations suggested filename is updated when selected group is changed, but only if the user did not edit the filename manually.

### Testing steps

1. Open the export annotations panel, and observe suggested filename matches selected group.
2. Change to another group with annotations, and verify suggested filename changed to include new group's name.
3. Edit the filename, and change again to previous group. Now the name should have been kept as is.

### TODO
